pkg/api/v1beta1/awspcaissuer_types.go

```diff
@@ -36,7 +36,15 @@ type AWSPCAIssuerSpec struct {
 	Region string `json:"region,omitempty"`
 	// Needs to be specified if you want to authorize with AWS using an access and secret key
 	// +optional
-	SecretRef v1.SecretReference `json:"secretRef,omitempty"`
+	SecretRef SecretReference `json:"secretRef,omitempty"`
+}
+
+type SecretReference struct {
```

Comment:
nit: Please rename to reflect that this struct is specifically for AWS access identifiers